src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
This comment is food for thought... 
Take a look and read on this document. Most of the react apps have src/ and public/ folder. index.html and favicon.ico are all put in the public/ folder. 
Read the following article... 
https://facebook.github.io/create-react-app/docs/using-the-public-folder